reference/docs-conceptual/components/vscode/using-vscode.md

```diff
@@ -210,89 +204,84 @@ last setting):
-set a breakpoint on a line, press <kbd>F9</kbd>, and then press <kbd>F5</kbd> to start debugging. You
-should see the Debug actions pane appear which allows you to break into the debugger, step, resume,
-and stop debugging.
+In Visual Studio Code version 1.9 (or higher), you can debug PowerShell scripts without opening the
+folder that contains the PowerShell script.
 
-### Workspace debugging
+1. Open the PowerShell script file with **File > Open File...**
+1. Set a breakpoint - select a line then press <kbd>F9</kbd>
+1. Press <kbd>F5</kbd> to start debugging
 
-Workspace debugging refers to debugging in the context of a folder that you've opened in Visual
-Studio Code from the **File** menu using **Open Folder...**. The folder you open is typically your
-PowerShell project folder and/or the root of your Git repository.
+You should see the Debug actions pane appear which allows you to break into the debugger,
+step, resume, and stop debugging.
 
-Even in this mode, you can start debugging the currently selected PowerShell script by pressing
-<kbd>F5</kbd>. However, workspace debugging allows you to define multiple debug configurations other
-than just debugging the currently open file. We'll get to those in a moment.
+### Workspace debugging
 
-Follow these steps to create your debug configuration file:
+Workspace debugging refers to debugging in the context of a folder that you've opened from the
+**File** menu using **Open Folder...**. The folder you open is typically your PowerShell project
+folder or the root of your Git repository. Workspace debugging allows you to define multiple debug
+configurations other than just debugging the currently open file.
+
+Follow these steps to create a debug configuration file:
 
   1. Open the **Debug** view on Windows or Linux by pressing
      <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>D</kbd>. On macOS, press
      <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>D</kbd>.
-  1. Click the "create a launch.json file" link.
-  1. Visual Studio Code prompts you to **Select Environment**. Choose **PowerShell**.
-  1. Lastly, choose the type of debugging you'd like to use:
+  1. Click the **create a launch.json file** link.
+  1. From the **Select Environment** prompt, choose **PowerShell**.
+  1. Choose the type of debugging you'd like to use:
 
-- **Launch Current File** - Launch and debug the file in the currently active editor window
-- **Launch Script** - Launch and debug the specified file or command
-- **Interactive Session** - Debug commands executed from the Integrated Console
-- **Attach** - Attach the debugger to a running PowerShell Host Process
+     - **Launch Current File** - Launch and debug the file in the currently active editor window
+     - **Launch Script** - Launch and debug the specified file or command
+     - **Interactive Session** - Debug commands executed from the Integrated Console
+     - **Attach** - Attach the debugger to a running PowerShell Host Process
```

Comment:
I'm not sure why the numbered list is indented, and it caused the sub bullets to need bonus indentation.